modules/gcp_nodepool/gcp-nodepool.md

```diff
@@ -29,4 +29,28 @@ existing roles attached there should be no problem.
 
 THIS SERVICE ACCOUNT IS NOT THE ONE USED BY YOUR CONTAINERS RUNNING IN THE CLUSTER-- Opta handles creating appropriate
 service accounts for each K8s service, but for any non-opta managed workloads in the cluster, please refer to this
-[GCP documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
\ No newline at end of file
+[GCP documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
+
+## Taints
+
+Opta now gives you the option of adding [taints](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
+to the nodes created in this nodepool. The official documentation gives an excellent detailed summary, but in short
+one can use taints to stop workloads from running in said nodes unless they have a matching toleration. Simply provide 
+a list of such taints as inputs like so:
+```yaml
+  - type: gcp-nodepool
+    name: nodepool1
+    min_nodes: 1
+    max_nodes: 3
+    taints:
+      - key: blah1
+        value: baloney
+        effect: "NO_EXECUTE"
+      - key: blah2
```

Comment:
Did you try adding/removing taints from a pre-existing nodepool? Does adding/removing taints to pre-existing node pools recreate them (slow)?